### ISSUE TITLE
Failed on Step 3 $ atomic

### ISSUE THREAD

**Issue by yasirmturk** [AtomicReact/AtomicReact#1]
1. `npm install atomicreact -g`
2. `npm install atomicreact --save`
3. `atomic`
```
{ Error: Command failed: node AtomicReact_run.js
fs.js:646
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^

Error: ENOENT: no such file or directory, open '/Users/punchee/Documents/personal/learning/AtomicReact/package.json'
    at Object.fs.openSync (fs.js:646:18)
    at Object.fs.readFileSync (fs.js:551:33)
    at new Atomic (/Users/punchee/Documents/personal/learning/AtomicReact/node_modules/atomicreact/Atomic.js:9:27)
    at Object.<anonymous> (/Users/punchee/Documents/personal/learning/AtomicReact/AtomicReact_run.js:8:16)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Function.Module.runMain (module.js:693:10)

    at ChildProcess.exithandler (child_process.js:275:12)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at maybeClose (internal/child_process.js:925:16)
    at Socket.stream.socket.on (internal/child_process.js:346:11)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at Pipe._handle.close [as _onclose] (net.js:557:12)
  killed: false,
  code: 1,
  signal: null,
  cmd: 'node AtomicReact_run.js' }
```

**Comment by Guihgo** [comment]
Did you create your package.json? 

type it:
```
npm init
```
if any question in this part, checkout this link: [https://docs.npmjs.com/creating-a-package-json-file](https://docs.npmjs.com/creating-a-package-json-file)

 and run again:

```
Atomic
```

**Comment by yasirmturk** [comment]
Alright, may be you should update the Getting started guide to:
`npm init && npm install atomicreact --save`

**Comment by Guihgo** [comment]
Fixed on commit: [b55a671c41f778b7286d3bc7aee19f0ad1df9df1](https://github.com/AtomicReact/AtomicReact/commit/b55a671c41f778b7286d3bc7aee19f0ad1df9df1)

Thanks @yasirmturk
